vector/src/main/java/im/vector/app/features/settings/VectorPreferences.kt

```diff
@@ -1001,7 +1001,9 @@ class VectorPreferences @Inject constructor(private val context: Context) {
     }
 
     fun labsRenderLocationsInTimeline(): Boolean {
-        return defaultPrefs.getBoolean(SETTINGS_LABS_RENDER_LOCATIONS_IN_TIMELINE, true)
+        // Tchap: Hide Location sharing
+//        return defaultPrefs.getBoolean(SETTINGS_LABS_RENDER_LOCATIONS_IN_TIMELINE, true)
+        return BuildConfig.SHOW_LOCATION_SHARING
```

Comment:
We can keep it as it is a lab section and only used to display the shared location in the timeline, not a flag for activating the sharing feature itself

Reply:
I will hide it because this is not working if we put an empty API key for MapTiler.